README.md

```diff
@@ -1,25 +1,35 @@
-[![Build status](https://travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
+# r0
+
+[![Build status](https://api.travis-ci.org/rust-embedded/r0.svg?branch=master)](https://travis-ci.org/rust-embedded/r0)
 [![crates.io](https://img.shields.io/crates/d/r0.svg)](https://crates.io/crates/r0)
 [![crates.io](https://img.shields.io/crates/v/r0.svg)](https://crates.io/crates/r0)
 
-# `r0`
+Memory initialization code **crt0** written in Rust. This crate is for bare metal systems where there is no ELF loader or OS to take care of initializing RAM for the program. Is also provides an API to add «life before main» functionality to bare metal systems.
```

Comment:
This is not a C runtime, which is why "crt0" is in quotes in the lib.rs documentation.

Also, all the life before main stuff has been removed.

Reply:
> all the life before main stuff has been removed.

Docs does not say anything about that.

> This is not a C runtime, which is why "crt0" is in quotes in the lib.rs documentation.
I will add quotes

Reply:
https://github.com/rust-embedded/r0/blob/master/CHANGELOG.md#unreleased

Reply:
done